adapters/repos/db/vector/hnsw/delete_test.go

```diff
@@ -951,18 +951,18 @@ func TestDelete_ResetLockDoesNotLockForever(t *testing.T) {
 			require.Nil(t, err)
 			require.True(t, ok)
 		}()
-		ellapsed := time.Duration(0)
+		elapsed := time.Duration(0)
 		go func() {
 			defer wg.Done()
 			time.Sleep(time.Millisecond * 100)
 			starting := time.Now()
 			err := vectorIndex.Delete(vectorIndex.entryPointID)
-			ellapsed = time.Since(starting)
+			elapsed = time.Since(starting)
 			require.Nil(t, err)
 		}()
 		wg.Wait()
-		fmt.Println(ellapsed.Milliseconds())
-		assert.LessOrEqual(t, ellapsed.Milliseconds(), int64(10))
+		fmt.Printf("ms elapsed: %d\n", elapsed.Milliseconds())
+		assert.LessOrEqual(t, elapsed.Milliseconds(), int64(15))
```

Comment:
@abdelr we have seen this test flake quite a bit with elapsed ms of `11`, `13`, and so forth. Haven't seen an instance where it is higher than 13, so I changed this max elapsed value to `15`

Reply:
I think that Abdel made a [PR](https://github.com/weaviate/weaviate/pull/5985/files) with a fix for this test targeted for `stable/v1.24`

Reply:
Ah very nice. Would it make sense to keep this, and allow it to be overridden with the merge-forward? This way the improved logging can stay?

Reply:
let's do that!